pkg/docker/config/config.go

```diff
@@ -719,3 +742,14 @@ func normalizeRegistry(registry string) string {
 	}
 	return normalized
 }
+
+// isNamespaced returns true if the provided registry contains one or more
+// subpaths.
+func isNamespaced(registry string) bool {
+	stripped := stripScheme(registry)
+
+	// don't take trailling slashes into account
+	stripped = strings.TrimSuffix(stripped, "/")
```

Comment:
We are not going to find entries with trailing slashes, so we should not allow writing them either.

Reply:
Alright, removed that additional modification.